src/routes/auth/index.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react'
-import { Route } from 'react-router-dom'
+import { Route, Switch } from 'react-router-dom'
 import Grid from 'material-ui/Grid'
 import SignUpRoute from 'routes/auth/SignUp'
+import SignInRoute from 'routes/auth/SignIn'
 
 const AuthRoute = ({ match }) => (
   <Grid container>
     <Grid item xs={12}>
-      <Route path={match.path + '/sign-up'} component={SignUpRoute} />
+      <Switch>
+        <Route path={match.path + '/sign-up'} component={SignUpRoute} />
```

Comment:
Please! Replace it with this \`${match.path}/sign-up\`

Reply:
@sergey-moiseyenko pls check once again